src/chai/ChaiManager.hpp

```diff
@@ -0,0 +1,28 @@
+//////////////////////////////////////////////////////////////////////////////
```

Comment:
I'm not sure I see the use for having this as a separate header. I'd prefer this change be reverted.